starter-code/css/style.css

```diff
@@ -0,0 +1,95 @@
+.navbar-brand {
+    font-size: 30px;
+    font-weight: bold;
+    color: black;
+}
+
+.navbar-nav{
+  padding-left: 166px;
+}
+
+.navbar-nav li a {
+    color: grey;
+}
+
+.navbar-nav li.active a {
+  color: #f4473a;
+}
+
+.first-slide{
+  width: 100%;
+  filter: grayscale(100%);
+}
+
+.carousel-caption h1{
+  text-transform: uppercase;
+}
+
+.carousel-caption p{
+  font-size: 20px;
+}
+
+.carousel-control.left,
+.carousel-control.right{
+  background-image: none;
+}
+
+.btn-primary{
+  background-color: rgba(255, 255, 255, 0.5);
+  border-color: white;
+  border-radius: 0px;
+  text-transform: uppercase;
+}
+
+.marketing{
+  background-color: #ed564b;
+  color: white;
+  text-align: center;
+  padding: 40px;
+}
+
+.col-lg-4 .glyphicon{
+  font-size: 20px;
+  padding: 20px;
+  border: 6px solid white;
+  border-radius: 100px;
+}
+
+
+.projects img, .articles img{
+  width: 100%;
+  margin: 25px 0px 15px 0px;
+}
+
+div.projects h2, .articles h2{
+  font-size: 24px;
+  color: #000;
+  font-weight: bold;
+  margin-bottom: 0px;
+}
+
+div.projects h3, .articles h3 {
+    font-size: 15px;
+    font-weight: bold;
+    border-bottom: 1px solid #dfdfdf;
+    color: #aaa;
+    padding-bottom: 20px;
+    margin: 0px;
+}
+
+.footer{
+  background-color: #3b3b3b;
+  color: white;
+  width: 1280px;
```

Comment:
On my larger screen the footer cuts off at a certain point. Remove the set width or set it to 100% so that it takes up the entire length of the viewport of the device that is rendering it. 

Also, the width of the footer or any other block element has 100% width set by default so there is no need to explicitly set it. You usually would only set the height size of the footer to give it the height.